docker/shared-build-env.dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM vimc/node-docker-openjdk:master
```

Comment:
This is mostly just copied from its previous location but github hasn't worked that out